### PR TITLE
libomp 15.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  CMake Error: The source directory "/tmp/libomp-20220919-36577-1wdezal/openmp-15.0.0.src/openmp-15.0.0.src" does not exist.
  Specify --help for usage, or press the help button on the CMake GUI.
```

**Comment by chenrui333** [comment]
relates to https://github.com/llvm/llvm-project/issues/57579

File: Formula/libomp.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libomp < Formula
   desc "LLVM's OpenMP runtime library"
   homepage "https://openmp.llvm.org/"
-  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.6/openmp-14.0.6.src.tar.xz"
-  sha256 "4f731ff202add030d9d68d4c6daabd91d3aeed9812e6a5b4968815cfdff0eb1f"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.0/openmp-15.0.0.src.tar.xz"
+  sha256 "7ee0d1c6a0426a34f102aa5c3a2c307807a4885ce347cb51fbc7e5effd6891c6"
```
Comment (carlocab) [inline Formula/libomp.rb:5]: You can use the original tarball if you download the `cmake` artefact too

Comment (chenrui333) [reply Formula/libomp.rb:5]: like this?

```diff
-  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.0/openmp-15.0.0.src.tar.xz"
-  sha256 "7ee0d1c6a0426a34f102aa5c3a2c307807a4885ce347cb51fbc7e5effd6891c6"
+  url "https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-15.0.0.tar.gz"
+  sha256 "36d83cd84e1caf2bcfda1669c029e2b949adb9860cff01e7d3246ac2348b11ae"
```

**Pull request closed without merging**